pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
```

Comment:
This is how it's done in OpenCV, which has a similar build system and also relies on boolean conditions to determine appropriate CMake flags. While those CMake flags aren't included in this initial push, I'm making a PR to include them. If you'd like, I can include them in this PR (I'll squash it)